### PR TITLE
Make tests runnable in all API levels

### PR DESCRIPTION
**Pull request by imurchie**
So we can begin to test fully.

### CODE REVIEW

File: test/functional/setup.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const avdName = process.env.ANDROID_AVD || 'NEXUS_S_18_X86';
+const platformVersion = process.env.PLATFORM_VERSION || '4.3';
+
+let apiLevel = process.env.API_LEVEL;
+if (!apiLevel) {
+  // https://developer.android.com/guide/topics/manifest/uses-sdk-element.html
+  switch (parseFloat(platformVersion)) {
```
Comment (jlipps) [inline test/functional/setup.js]: this seems a little verbose, maybe something lke:

```js
if (!apiLevel) {
  let levelMap = {
    '4.1': '16',
    '4.2': '17',
    '4.3': '18',
    '4.4': '19',
    '5': '21',
    '5.1': '22',
    '6': '23',
    '7': '24',
    '7.1': '25',
  };
  apiLevel = levelMap[parseFloat(platformVersion).toString()];
}
```

?

File: test/functional/apk-utils-e2e-specs.js [hunk 2]
```diff
@@ -95,18 +95,4 @@ describe('apk utils', function () {
     let {apkStrings} = await adb.extractStringsFromApk(contactManagerPath, null, '/tmp');
     apkStrings.save.should.equal('Save');
   });
-  it('should get device language and country', async () => {
```
Comment (jlipps) [inline test/functional/apk-utils-e2e-specs.js]: these tests are duplicated above?

Comment (imurchie) [inline test/functional/apk-utils-e2e-specs.js]: Yes.

File: test/functional/setup.js (hunk 1 above)
Comment (imurchie) [inline test/functional/setup.js]: Makes sense. I've updated a little differently, but same.

**Pull request merged**